lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -481,7 +482,7 @@ struct ElementwiseInlineAsmOpConversion
           op.getPackedElement() - numElemsPerThread % op.getPackedElement();
       for (auto &operands : unpackedOperands) {
         for (int i = 0; i < numPaddedValue; i++) {
-          operands.push_back(operands[0]);
+          operands.push_back(undef(operands[0].getType()));
```

Comment:
Nit: outdated comment above ("first value of the operand")

Reply:
Good catch, thanks!